### ISSUE TITLE
Does not work at all

### ISSUE THREAD

**Issue by Gaibhne** [Tarik02/vscode-php-imports#1]
VSCode: 1.60.1, php-imports v0.3.0.

Just outright doesn't do anything. No changes, no error message, just a hover box saying "Imports are already formatted", which they are not, and adding newlines or changing orders does also not result in the plugin doing anything at all.

**Comment by Tarik02** [comment]
Hello. Can you please post the code you are having problems with?

**Comment by Gaibhne** [comment]
Sure:

```php
<?php

declare(strict_types=1);

namespace AppBundle\Command;

use Symfony\Component\Console\Command\Command;
use Symfony\Component\Console\Input\InputInterface;
use Company\Json\JsonHandler;
use Symfony\Component\Console\Input\InputOption;

use Symfony\Component\Console\Output\OutputInterface;

// class definition
```

As you can see, the imports are out of order, contain an unnecessary space and one of the imports isn't used. None of it is affected or fixed.

**Comment by Tarik02** [comment]
> Sure:
> 
> ```
> <?php
> 
> declare(strict_types=1);
> 
> namespace AppBundle\Command;
> 
> use Symfony\Component\Console\Command\Command;
> use Symfony\Component\Console\Input\InputInterface;
> use Company\Json\JsonHandler;
> use Symfony\Component\Console\Input\InputOption;
> 
> use Symfony\Component\Console\Output\OutputInterface;
> 
> // class definition
> ```
> 
> As you can see, the imports are out of order, contain an unnecessary space and one of the imports isn't used. None of it is affected or fixed.

The problem is caused by `declare(strict_types=1);`. Currently php-imports parser does not parse them. This is considered as a bug, I'll fix it.

**Comment by Tarik02** [comment]
Fixed with version 0.4.0.
